builtin/game/falling.lua

```diff
@@ -42,8 +42,7 @@ core.register_entity(":__builtin:falling_node", {
 
 	on_step = function(self, dtime)
 		 -- Set gravity
-		local acceleration = self.object:getacceleration()
-		if not vector.equals(acceleration, {x = 0, y = -10, z = 0}) then
+		if self.object:getacceleration().y ~= -10 then
 			self.object:setacceleration({x = 0, y = -10, z = 0})
```

Comment:
If possible I'd rather keep x and z set to 0, so that tnt doesn't blow falling nodes out of the screen, the 0 stops it going TOO far and makes them fall vertically.